### ISSUE TITLE
Unable use `GetOrder` if swish payment type is used

### ISSUE THREAD

**Issue by Flybring** [sveawebpay/svea-sdk-dotnet#111]
If `SWISH` PaymentType is used during checkout I'm unable to fetch the order after the payment has been completed.

```
The JSON value could not be converted to Svea.WebPay.SDK.CheckoutApi.Data. Path: $.PaymentType | LineNumber: 0 | BytePositionInLine: 3349.

   at System.Text.Json.ThrowHelper.ThrowJsonException(String message)
   at System.Text.Json.Serialization.Converters.EnumConverter`1.ReadWithQuotes(Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.EnumConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.Converters.NullableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryReadAsObject(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state, Object& value)
   at System.Text.Json.Serialization.Converters.LargeObjectWithParameterizedConstructorConverter`1.ReadAndCacheConstructorArgument(ReadStack& state, Utf8JsonReader& reader, JsonParameterInfo jsonParameterInfo)
   at System.Text.Json.Serialization.Converters.ObjectWithParameterizedConstructorConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadCore[TValue](JsonConverter jsonConverter, Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadCore[TValue](Utf8JsonReader& reader, Type returnType, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, Type returnType, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, JsonSerializerOptions options)
   at Svea.WebPay.SDK.SveaHttpClient.<SendHttpRequestAndProcessHttpResponse>d__10`1.MoveNext()
```

**Comment by lounge** [comment]
@Flybring What version of sdk are you using? This works fine in latest release 3.1.0

**Comment by lounge** [comment]
@Flybring Please give steps to reproduce.

**Comment by Flybring** [comment]
Here you go @lounge 

1. Create order with `client.Checkout.CreateOrder`
2. Render `Gui.Snippet`
3. Select Swish payment
4. Pay trough Swish app
5. Fetch order with `client.Checkout.GetOrder` after payment is done

I realized there is an `GetOrder` method trough `PaymentAdmin` as well. I do not get any error from that method.

**Comment by lounge** [comment]
@Flybring This is because SWISH_PF is missing from SDK. There is only support for SWISH as of now. Will be fixed in next release

**Comment by Flybring** [comment]
@lounge Any ETA on the next release?

**Comment by lounge** [comment]
@Flybring We are planning on a new release this or next week with a fix.

**Comment by lounge** [comment]
@Flybring This is fixed in release 4.0.0
